Riku/src/Frontend/FrontendState.cpp

```diff
@@ -44,6 +45,11 @@ int FrontendState::getPlayerOnMove()
 	return logic.getInfo<IntResponse>("player_on_move")->get();
 }
 
+std::vector<std::string> FrontendState::getTileObjectGui(int mapX, int mapY)
```

Comment:
jeśli getTileObjectGui zwraca opcjo do wykorzystania w choseGuiOption to bardziej intuicyjna byłaby nazwa w stylu getGuiOptions

Reply:
Masz rację